sdk/nodejs/src/bucketing.ts

```diff
@@ -5,67 +5,49 @@ import {
     DevCycleServerSDKOptions,
 } from '@devcycle/types'
 
-let Bucketing: Exports | null
-let InstantiatePromise: Promise<Exports> | null
-
 export const importBucketingLib = async ({
     logger,
     options,
 }: {
     logger?: DVCLogger
     options?: DevCycleServerSDKOptions
-} = {}): Promise<void> => {
-    if (InstantiatePromise) {
-        await InstantiatePromise
-        return
-    }
+} = {}): Promise<[Exports, NodeJS.Timer | undefined]> => {
```

Comment:
I think we should rename the `Exports` import to something clearer like `WASMBucketingLib` or something like that.